Sources/TuistKit/Services/ScaffoldService.swift

```diff
@@ -20,7 +21,7 @@ enum ScaffoldServiceError: FatalError, Equatable {
     var description: String {
         switch self {
         case let .templateNotFound(template):
-            return "Could not find template \(template). Make sure it exists at Tuist/Templates/\(template)"
+            return "Could not find template \(template)."
```

Comment:
I'd still provide information about where the template is expected. Rather than deleting the part about local template, I'd extend it with that we were not able to find it in the plugins as well.